common/locator.go

```diff
@@ -31,21 +31,17 @@ func NewLocator(ctx context.Context, selector string, f *Frame, l *log.Logger) *
 }
 
 // Click on an element using locator's selector with strict mode on.
-func (l *Locator) Click(opts goja.Value) {
+func (l *Locator) Click(opts goja.Value) error {
 	l.log.Debugf("Locator:Click", "fid:%s furl:%q sel:%q opts:%+v", l.frame.ID(), l.frame.URL(), l.selector, opts)
 
-	var err error
-	defer func() { panicOrSlowMo(l.ctx, err) }()
```

Comment:
Do we not need `slowMo` for `click`?

Reply:
Nice catch, let me add it back.

Reply:
Resolved in https://github.com/grafana/xk6-browser/blob/ab096eb7a09e13646df318a69ed8abb0e6514761/common/locator.go#L44-L46.